website/blog/2022-09-28-analyst-to-ae.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Analysts make the best analytics engineers"
+description: "You often hear of the disgruntled analyst becoming the analytics engineer..and loving it? In this article, Brittany walks through what makes analysts ripe for best pracitice analytics engineering work."
+slug: analysts-make-the-best-aes
+authors: [brittany_krauth]
+tags: [analytics craft]
+hide_table_of_contents: false
+
+date: 2022-09-29
+is_featured: true
+---
+
+When you were in grade school, did you ever play the “Telephone Game”? The first person would whisper a word to the second person, who would then whisper a word to the third person, and so on and so on. At the end of the line, the final person would loudly announce the word that they heard, and alas! It would have morphed into a new word completely incomprehensible from the original word. That’s how life feels without an Analytics Engineer on your team.
```

Comment:
> That’s how life feels without an Analytics Engineer on your team.

Is this supposed to refer to a specific person we're naming  "Analytics Engineer"? — kind of like how a contract does it with "Landlord" and "Tenant"?

It doesn't seem that way from my read, but I could be wrong! If not, this and all the other job titles should be in lowercase instead of uppercase

Reply:
I think that most instances of Analyst + Analytics Engineer are capitalized (that's what Brittany wanted)